src/styles/Lesson7.scss

```diff
@@ -0,0 +1,8 @@
+@import './global';
+
+// lesson7slides
+// =======================================
+
+.lesson7slides {
+  width: 55%;
```

Comment:
Is there a reason for a width of only 55%? I would think it should take up more space.
Also, if you want to take a stab at fixing this image aspect ratio issue for different screen sizes, you can. This can be a separate ticket though.
<img width="1232" alt="image" src="https://github.com/uclaacm/parcel-pointers/assets/78634083/6e48afb1-1a46-468b-958c-c8da0aaa37a7">

Reply:
When there was no width the image would just be small unless I added the 55%, I think the image aspect ratio should be a separate ticket for now.